vach/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vach"
-version = "0.1.3"
+version = "0.1.5"
```

Comment:
Shouldn't we only update the version when we release a new version on crates.io?

Reply:
I plan on updating the spec, this is not supposed to be here